src/js/hamburger.js

```diff
@@ -0,0 +1,8 @@
+const btn = document.querySelector(".jsNavDropBtn");
```

Comment:
to make it more isolated and well structured, the code should be placed to the module pattern and the document should be passed as a parameter. in this case you have a global `btn` and other variables